ReactAndroid/src/main/java/com/facebook/react/bridge/ReactBridge.java

```diff
@@ -82,6 +82,7 @@ private native void initialize(
   public native void callFunction(int moduleId, int methodId, NativeArray arguments);
   public native void invokeCallback(int callbackID, NativeArray arguments);
   public native void setGlobalVariable(String propertyName, String jsonEncodedArgument);
+  public native long getJavaScriptContext();
```

Comment:
Name something convincingly scary like getJavaScriptContextNativePtrExperimental and document that this API will go away soon.